xml/containerized-kubernetes.xml

```diff
@@ -62,23 +60,48 @@
  </sect1>
  <sect1 xml:id="kube-prereq">
   <title>Prerequisites</title>
-
   <para>
-   Before you start deploying, you need to have:
+   The minimum requirements and prerequisites to deploy &productname; &productnumber;
+   on top of &casp; 4 &kube; cluster are as follows:
   </para>
-
   <itemizedlist>
    <listitem>
     <para>
-     A running &casp; 4 cluster.
+     A running &casp; 4 cluster. You need to have an account with a
+     &casp; subscription. You can activate a 60-day free evaluation
+     here <link xlink:href="https://www.suse.com/products/caas-platform/download/MkpwEt3Ub98~/?campaign_name=Eval:_CaaSP_4"/>.
     </para>
    </listitem>
    <listitem>
     <para>
-     &casp; 4 worker nodes with a number of extra disks attached as storage for
-     the &ceph; cluster.
+     At least three &casp; worker nodes, with at least one additional disk attached to
+     each worker node as storage for the OSD. We recommend four &casp; worker nodes.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       At least one OSD per worker node, with a minimum disk size of 5&nbsp;GB.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Access to &productname; &productnumber;. You can get a trial
+       subscription from here <link xlink:href="https://www.suse.com/products/suse-enterprise-storage/download/"/>.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Access to a workstation that has access via <literal>kubectl</literal> to the
```

Comment:
Suggestion:
"Access to a workstation that has access to the cluster via kubectl" -- that word order makes more sense to me. What do you think?

The master node as the workstation? Really? It can orchestrate itself? I did not know that was possible. Does it not also mean the master must run X.11 and so on? That does not sound like a good idea, but I defer to the experts.

Reply:
@denisok ?

Reply:
I don't think we should be picky on this, especially for tech preview. fine I am.